bin/run_gosec

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
```

Comment:
as a follow-on to this PR, does it make sense to include this as a bash-lib script? cc @hughsaunders